dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,173 @@
+import com.dotmarketing.beans.Inode;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.fixtask.FixTask;
+import com.dotmarketing.portlets.cmsmaintenance.ajax.FixAssetsProcessStatus;
+import com.dotmarketing.portlets.cmsmaintenance.factories.CMSMaintenanceFactory;
+import com.dotmarketing.util.ConfigUtils;
+import com.dotmarketing.util.Logger;
+
+
+public class FixTask00095DeleteOrphanRelationships implements FixTask{
+
+    private List<Map<String, String>> modifiedData = new ArrayList<Map<String, String>>();
+
+    /** Lookup invalid inodes in Field table referencing the Structure table */
+    private static final String VERIFICATION_QUERY = "SELECT inode FROM relationship WHERE NOT EXISTS " +
+            "(SELECT * FROM structure WHERE relationship.parent_structure_inode = structure.inode) " +
+            " OR NOT EXISTS " +
+            "(SELECT * FROM structure WHERE relationship.child_structure_inode = structure.inode) ";
+
+    @Override
+    public List<Map<String, Object>> executeFix() throws DotDataException,
+            DotRuntimeException {
+        Logger.info(FixTask00095DeleteOrphanRelationships.class,
+                "Beginning DeleteOrphanRelationships");
+        List<Map<String, Object>> returnValue = new ArrayList<Map<String, Object>>();
+
+        if (!FixAssetsProcessStatus.getRunning()) {
+            HibernateUtil.startTransaction();
+            int total = 0;
+            try {
+                FixAssetsProcessStatus.startProgress();
+                FixAssetsProcessStatus
+                        .setDescription("task 95: DeleteOrphanRelationships");
+                DotConnect dc = new DotConnect();
+                dc.setSQL(VERIFICATION_QUERY);
+                modifiedData = dc.loadResults();
+                total = modifiedData != null ? modifiedData.size() : 0;
+                FixAssetsProcessStatus.setTotal(total);
+                getModifiedData();
+                if (total > 0) {
+                    try {
+                        HibernateUtil.startTransaction();
+                        cleanUpOrphanRelationships();
+                        
+                        HibernateUtil.commitTransaction();
+                        // Set the number of records that were fixed
+                        FixAssetsProcessStatus.setErrorsFixed(modifiedData.size());
+                    } catch (Exception e) {
+                        Logger.error(
+                                this,
+                                "Unable to clean orphaned content type fields.",
+                                e);
+                        HibernateUtil.rollbackTransaction();
+                        modifiedData.clear();
+                    }
+                    finally {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "finally" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)